docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Learners;
+using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class PFI_RegressionExample
+    {
+        public static void PFI_Regression()
+        {
+            // Download the dataset from github.com/dotnet/machinelearning.
+            // This will create a housing.txt file in the filesystem.
+            // You can open this file to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Step 1: Read the data as an IDataView.
+            // First, we define the reader: specify the data columns and where to find them in the text file.
+            // The data looks like this:
+            //     24.00  0.00632  18.00  2.310  0  0.5380  6.5750  65.20  4.0900  1  296.0  15.30
+            //     21.60  0.02731   0.00  7.070  0  0.4690  6.4210  78.90  4.9671  2  242.0  17.80
+            var reader = mlContext.Data.TextReader(new TextLoader.Arguments()
+                {
+                    Separator = "tab",
+                    HasHeader = true,
+                    Column = new[]
+                    {
+                        new TextLoader.Column("MedianHomeValue", DataKind.R4, 0),
```

Comment:
// read the first column (indexed by 0) in the data file

Reply:
Done.